src/python/psychrolib.py

```diff
@@ -172,6 +172,13 @@ def SetUnitSystem(Units: UnitSystem) -> None:
     else:
         PSYCHROLIB_TOLERANCE = 0.001
 
+    if has_numba:
```

Comment:
We need comments before this line, to explain to users why this section is needed and what it does.